### PR TITLE
Update discussion of entry priorities in constant tables

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -3309,19 +3309,17 @@ The contents of const tables can be queried by the client through a
...
 
-[^ConstEntriesPriorities]: This is not yet explicit in the P4~16~
-    language specification, but will become so if this or a similar
-    clarification is added to it:
-    <https://github.com/p4lang/p4-spec/pull/1259>
+When a priority value is required (&eg; for tables including `RANGE`,
+`TERNARY` or `OPTIONAL` matches), it is inferred based on the order in
+which entries appear in the table declaration.  As of August 2023, the
+open source `p4c` compiler always assigns entry priority values in
+constant tables starting at 1 for the first entry and incrementing the
```
Comment (jonathan-dilorenzo) [inline docs/v1/P4Runtime-Spec.mdk:3318]: Interesting. To be clear, this means that the last entry has the highest priority atm?

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk:3318]: Hmmm.   This text may have always been wrong (or unclear), or perhaps my recent edits have distorted its meaning.

The current implementation is that earlier entries textually in the source code "win" over later entries textually.  Let me add another commit to this PR to correct this.

Good catch!  I can only make the lame excuse that I had not finished my morning coffee yet while editing this PR.

Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk:3318]: Haha, personally I consider that a fantastic excuse, so no worries at all ;).

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: I believe commit 2 is factually correct.

Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk]: LGTM

**Review by jonathan-dilorenzo — APPROVED** [review]

**Review by chrispsommers — APPROVED** [review]

**Pull request merged**